pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -1151,9 +1151,34 @@ func validateParamRef(pr *admissionregistration.ParamRef, fldPath *field.Path) f
 	if pr == nil {
 		return allErrors
 	}
-	for _, msg := range path.ValidatePathSegmentName(pr.Name, false) {
-		allErrors = append(allErrors, field.Invalid(fldPath.Child("name"), pr.Name, msg))
+
```

Comment:
make sure that one of `name` and `selector` is specified

Reply:
added and added validation test